lib/ClimaLandSimulations/src/Fluxnet/fluxnet_sites/US-MOz/US-MOz_parameters.jl

```diff
@@ -158,6 +158,7 @@ function photosynthesis_ozark(;
     ΔHΓstar = FT(37830),
     ΔHJmax = FT(43540),
     ΔHRd = FT(46390),
+    sif_parameters = SIFParameters{FT}(),
 )
     return FarquharParameters(
```

Comment:
do we need to pass sif_parameters here? I thought in our methods in CreateParametersExt we have a default which makes the sif_params and uses them

Reply:
We need it here because it calls the struct, not the function. 

Note that we plan to refactor ClimaLandSimulations (with the help of Gabriele), to have it better designed in general, with PFT framework, global simulations... and ready for ClimaCalibrate